modules/services/views/src/main/java/au/com/shiftyjelly/pocketcasts/views/helper/ShowNotesFormatter.kt

```diff
@@ -61,7 +64,10 @@ class ShowNotesFormatter(settings: Settings?, private val context: Context) {
             }
 
             addShowNotesHead(html)
-            html.append("<body>")
+
+            val isRtl = Locale.getDefault().layoutDirection == LayoutDirection.RTL
```

Comment:
I think this should be checking for the language of the show notes, not of the device. Otherwise if you are subscribed to podcasts in different languages, half of them will always be wrong

Reply:
Good point! Using `Bidi` instead to determine the direction of the show notes and switch the `WebView` to rtl, should match the behaviour of the `TextView` which shows the title rtl even when the device locale is set to a language that is not rtl 😃 

Cheers!